src/users/components/RemoveMemberOverlay.tsx

```diff
@@ -0,0 +1,147 @@
+// Libraries
+import React, {FC, useContext, useState} from 'react'
+import {useSelector} from 'react-redux'
+
+import {
+  Alert,
+  AlignItems,
+  Button,
+  ComponentColor,
+  ComponentStatus,
+  Dropdown,
+  FlexBox,
+  FlexDirection,
+  Form,
+  IconFont,
+  Overlay,
+  RemoteDataState,
+  SpinnerContainer,
+  TechnoSpinner,
+} from '@influxdata/clockface'
+
+// Contexts
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+// Constants
+import {CLOUD_URL} from 'src/shared/constants'
+
+// Selectors
+import {selectUser} from 'src/identity/selectors'
+
+// Types
+import {User} from 'src/client/unityRoutes'
+import {UsersContextType} from 'src/users/context/users'
+
+export interface RemoveMemberOverlayParams {
+  removeUser: UsersContextType['removeUser']
+  users: User[]
+  userToRemove: User
+}
+
+export const RemoveMemberOverlay: FC = () => {
+  const {onClose, params} = useContext(OverlayContext)
+  const {removeUser, users, userToRemove} = params
+  const otherUsers = users.filter((user: User) => user.id !== userToRemove.id)
```

Comment:
The _current_ user can't be an option as far as who should receive the idpe assets.